packages/zwave-js/src/lib/driver/UpdateConfig.ts

```diff
@@ -50,6 +52,14 @@ export async function checkForConfigUpdates(
  * Installs the update for @zwave-js/config with the given version.
  */
 export async function installConfigUpdate(newVersion: string): Promise<void> {
+	if (_installConfigLock) {
+		throw new ZWaveError(
+			`Config update failed: another installation is in progress`,
+			ZWaveErrorCodes.Config_Update_InstallFailed,
+		);
```

Comment:
Not sure if I want to throw an error here. In other locations I've indicated a busy status by returning `false` and `true` if something was done.

Reply:
Ok so should I return false here?